source/common/types/cardanoNode.types.js

```diff
@@ -24,9 +42,64 @@ export const CardanoNodeStates: {
   ERRORED: 'errored',
 };
 
-export type TlsConfig = {
-  port: number,
-  ca: Uint8Array,
-  cert: Uint8Array,
-  key: Uint8Array,
+export type CardanoPidOptions = (
+  'mainnet-PREVIOUS-CARDANO-PID' |
+  'staging-PREVIOUS-CARDANO-PID' |
+  'testnet-PREVIOUS-CARDANO-PID' |
+  'development-PREVIOUS-CARDANO-PID' |
+  string
+);
+
+export type DaedalusPidOptions = (
+  'mainnet-PREVIOUS-DAEDALUS-PID' |
+  'staging-PREVIOUS-DAEDALUS-PID' |
+  'testnet-PREVIOUS-DAEDALUS-PID' |
+  'development-PREVIOUS-DAEDALUS-PID' |
+  string
+);
+
+export type CardanoNodeStorageKeys = {
+  PREVIOUS_CARDANO_PID: CardanoPidOptions,
+  PREVIOUS_DAEDALUS_PID: DaedalusPidOptions
+};
+
+// TODO: Add more options when the exact process names are established
+export type DaedalusProcessNames = (
+  'Electron'
+);
+
+// TODO: Determine if the cardano-node can be named anything else
```

Comment:
@cleverca22 same thing here - is the Cardano Node process always called `cardano-node` on all of the platforms?

Reply:
@cleverca22 this is still important to know!

Reply:
on windows it will be `cardano-node.exe` and on linux and darwin, just `cardano-node`

Reply:
Thanks @cleverca22!
@MarcusHurney please update the code to match this 🙏

Reply:
@nikolaglumac 
I've added it!